cardano-testnet/cardano-testnet.cabal

```diff
@@ -203,6 +203,7 @@ test-suite cardano-testnet-test
                         Cardano.Testnet.Test.Gov.ProposeNewConstitution
                         Cardano.Testnet.Test.Gov.ProposeNewConstitutionSPO
                         Cardano.Testnet.Test.Gov.GovActionTimeout
+                        Cardano.Testnet.Test.Gov.TransactionBuildWrongHash
```

Comment:
`Cardano.Testnet.Test.Gov.Transaction.HashMismatch` would be a better namespace.